README.md

```diff
@@ -316,8 +316,8 @@ Additional documentation for the Flexy Java application is available in the Ewon
 
 ### Installation
 Using FTP, transfer the Flexy Java application \(.jar\) and jvmrun files to the /usr/ directory of the Ewon. Then, reboot the Ewon. On application startup, a configuration file will automatically be created with default values.
-
-Those files are located in the /built-flexy-java-app/ directory of the release zip, found at [https://github.com/hms-networks/flexy-thingworx-connector/releases/latest](https://github.com/hms-networks/flexy-thingworx-connector/releases/latest). When building from source code, the jvmrun file is located in /scripts/, and the Flexy Java application \(.jar\) is located in /build/.
+- When installing from a zipped release, the Flexy Java application \(.jar\) and jvmrun files are located in the /built-flexy-java-app/ directory. 
+- When installing from the source code, the jvmrun file is located in /scripts/, and the Flexy Java application \(.jar\) is located in /build/ after compilation is run.
```

Comment:
"When installing from the source code" - This is kind of confusing. I prefer the original phrasing that referenced building from sources. 

Maybe something like "When building and installing from the source code"

Also should you clarify here what users should be doing this and why. It might be confusing to someone how to use github and they may think they need to do something with the source.

Reply:
Added more detail/clarification to the source code line